helm/hpcc/templates/thor.yaml

```diff
@@ -175,6 +176,7 @@ data:
       backoffLimit: 0
 {{- end }}
 
+  {{- $thorMgrJobNamePrefix := printf "%s-thormanager-" .name }}
```

Comment:
change pod name on line 184 too.

Reply:
I cannot set 
   kind: Job$
      metadata:$
         name: $thorMgrJobNamePrefix
<HPCC-Platform>/common/workunit/workunit.cpp only works for job pattern "thormanager-.*" not "thor1-thormanager-.*" if "thor1" is first target name as:
thor:
   \- name: thor1

For example, "kubectl delete ..."

I set "<target>-thormanager- " so the placement settings can only be applied to this target job not the other such as "thor2". We can discuss this more in flowdock